### DISCUSSION TITLE
RV280 no roms

### DISCUSSION THREAD

**Discussion by Ivobaz** [eduardofilo/RG350_adam_image#312] [General]
Hello,
I follewed the guide and have done it several times and always the roms dont apear in simple menu. I can see them in Commander app. If i tried to execute it gives the rror : " exec format error" .
Thank you

**Comment by eduardofilo** [comment]
You can't execute ROMs from Commander. That possibility is for executable files (OPK, sh). Try following this entry in FAQ:
https://github.com/eduardofilo/RG350_adam_image/wiki/En:-7.-FAQ#q10-why-dont-appear-in-simplemenu-the-roms-i-just-loaded-onto-the-external-card
